src/ubuntu-generic-diskimages/scripts/post-installation.sh

```diff
@@ -45,9 +51,9 @@ cp -r include/gem5 /usr/local/include/\
 
 # Build the library and binary
 pushd util/m5
-scons build/x86/out/m5
-cp build/x86/out/m5 /usr/local/bin/
-cp build/x86/out/libm5.a /usr/local/lib/
+scons build/${ISA}/out/m5
+cp build/${ISA}/out/m5 /usr/local/bin/
+cp build/${ISA}/out/libm5.a /usr/local/lib/
 popd
 popd
```

Comment:
It would be good to let people know what we're popping

```suggestion
popd   # util/m5
popd   # gem5
```